projects/1.16.1/assets/playerex/playerex/lang/zh_cn.json

```
@@ -0,0 +1,108 @@
+{
+	"playerex.container": "玩家属性",
+	"playerex.page.player_attributes": "属性",
+	"playerex.page.player_resistances": "抗性",
+	"playerex.config.client.enablegui": "启用/禁用自定义GUI。",
+	"playerex.config.common.constitution": "体质",
+	"playerex.config.common.strength": "力量",
+	"playerex.config.common.dexterity": "敏捷",
+	"playerex.config.common.intelligence": "智力",
+	"playerex.config.common.luckiness": "幸运",
+	"attribute.name.playerex.experience": "经验",
+	"attribute.name.playerex.level": "等级",
+	"attribute.name.playerex.skillpoints": "属性点",
+	"attribute.name.playerex.constitution": "体质",
+	"attribute.name.playerex.strength": "力量",
+	"attribute.name.playerex.dexterity": "敏捷",
+	"attribute.name.playerex.intelligence": "智力",
+	"attribute.name.playerex.luckiness": "幸运",
+	"attribute.name.playerex.health_regen": "生命恢复",
+	"attribute.name.playerex.health_regen_amp": "生命恢复倍率",
+	"attribute.name.playerex.damage_reduction": "伤害减免",
+	"attribute.name.playerex.fire_resistance": "火焰抗性",
+	"attribute.name.playerex.lava_resistance": "岩浆抗性",
+	"attribute.name.playerex.explosion_resistance": "爆炸抗性",
+	"attribute.name.playerex.falling_resistance": "摔落抗性",
+	"attribute.name.playerex.poison_resistance": "中毒抗性",
+	"attribute.name.playerex.wither_resistance": "凋零抗性",
+	"attribute.name.playerex.drowning_resistance": "溺水抗性",
+	"attribute.name.playerex.melee_crit_damage": "暴击伤害(近)",
+	"attribute.name.playerex.melee_crit_chance": "暴击机率(近)",
+	"attribute.name.playerex.evasion": "闪避",
+	"attribute.name.playerex.ranged_damage": "远程伤害",
+	"attribute.name.playerex.ranged_crit_damage": "暴击伤害(远)",
+	"attribute.name.playerex.ranged_crit_chance": "暴击机率(远)",
+	"attribute.name.playerex.lifesteal": "吸血",
+	"playerex.attribute.level": "等级: +%s (%s%s)",
+	"playerex.attribute.level.alt": "玩家等级,与附魔等级是分开的,每升级一级获得一点属性点.",
+	"playerex.attribute.skillpoints": "属性点: +%s",
+	"playerex.attribute.skillpoints.alt": "显示你还有多少属性点,用给增加下面那五个基础属性.",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"playerex.attribute.skillpoints.alt": "显示剩余的属性点数，属性点可用于增强下列的五种基础属性。",
```

过于口语 ，虽不算问题，但理应更为严谨。